fleetctl/status.go

```diff
@@ -82,7 +82,7 @@ func runStatusUnits(args []string) (exit int) {
 
 		cmd := fmt.Sprintf("systemctl status -l %s", name)
 		if exit := runCommand(cmd, uMap[name].MachineID); exit != 0 {
```

Comment:
Thanks for the fix; I think the better solution here would be to stop masking the return variable. So, change `if exit :=` to `if exit =`, then leave the break statement as it was. Would you mind making that change?